modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -167,7 +204,7 @@ private void doStartNode() {
             Uri gethLogUri = Uri.fromFile(logFile);
             try {
                 Log.d(TAG, "Attach to geth.log to instabug " + gethLogUri.getPath());
-                Instabug.setFileAttachment(gethLogUri, gethLogFileName);
+                //Instabug.setFileAttachment(gethLogUri, gethLogFileName);
```

Comment:
Same here, relationship to network switching?

Reply:
probably not, it's related to some intermediate testing